.github/CODEOWNERS

```diff
@@ -56,24 +55,25 @@
 /iam/                                  @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-infra 
 /kms/                                  @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-infra
 /securitycenter/                       @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-infra 
+/mediacdn/                             @justin-mp @GoogleCloudPlatform/dee-infra @GoogleCloudPlatform/go-samples-reviewers
```

Comment:
Please do not add individuals to CODEOWNERS.

Reply:
nit: please keep "go-samples-reviewers" as first in the list for tidiness reasons.

Reply:
I support the rule of not adding individuals to CODEOWNERS, but I wanted to make an exception in this case. We are assisting the product team with these samples therefore dee-infra is here plus TL from product. Creating a teamsynced group with just @justin-mp as a member seems to be an overkill.

I have switched the order as suggested.